### PR TITLE
	Cleaning up #ifdefs that break statements

### PR DESCRIPTION
**Pull request by RomeroMalaquias**

### CODE REVIEW

File: src/plugins/yajl/yajl_parse.c [hunk 1]
```diff
@@ -395,12 +396,11 @@ int elektraYajlGet(Plugin *handle ELEKTRA_UNUSED, KeySet *returned,
 		{
 			stat = yajl_parse(hand, fileData, rd);
 		}
-
-		if (stat != yajl_status_ok
+		test_status = (stat != yajl_status_ok);
```
Comment (markus2330) [inline src/plugins/yajl/yajl_parse.c]: variable could also be declared here

**Comment by markus2330** [comment]
Thanks for the contribution! Did you test if yajl1 still works?

**Comment by markus2330** [comment]
jenkins build please

**Comment by RomeroMalaquias** [comment]
Hi @markus2330, i didn't tested, sorry. 
If you approve that change and want the declaration of variable in another place, i can replace it.

**Comment by markus2330** [comment]
I agree that the change is an improvement (with variable declarations in either way, even though I prefer the declaration as late as possible). Testing it would be good though, I do not have yajl1 easily available anymore. Is there any specific reason you need the change, or is it just to check if your PRs are accepted for further contributions?

**Comment by RomeroMalaquias** [comment]
it's just to check if developers care about the discipline of conditional annotations

**Comment by markus2330** [comment]
Interesting study. Was this checked and/or repaired by a tool? Seems to be correct. Would be nice if you can move the variable within the block.

**Comment by RomeroMalaquias** [comment]
We have an tool to point possible candidates to refactoring, but still don't have one to test.
Because of this, i look for refactoring that aren't much complex.

**Comment by markus2330** [comment]
Ok, thanks for the fix. Further improvements are welcome.

**Pull request merged**